.github/workflows/integration-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jdk: [11, 17]
+        jdk: [17]
```

Comment:
For this bulk test job, why not run these tests on both platforms?

Reply:
I upgrade the spring test framework, it not support jdk 11, when I do the the test, it failed, raise an error a class version error. So I remove the jdk11, left jdk17
org.springframework.kafka:spring-kafka-test:3.0.3